src/gtl/components/tile-view/tile-view.html

```diff
@@ -1,22 +1,22 @@
 <div class="miq-tile-section">
   <div ng-if="tileCtrl.settings.isLoading" class="spinner spinner-lg"></div>
   <div class="row miq-pagination" ng-if="tileCtrl.rows && tileCtrl.rows.length !== 0">
-    <div ng-if="tileCtrl.rows.length !== 0" class="miq-select-all col-md-3 col-ld-3">
+    <div ng-if="tileCtrl.rows.length !== 0" class="miq-select-all col-md-3 col-ld-3 col-xs-3">
       <label>{{tileCtrl.settings.selectAllTitle}}: </label>
       <input type="checkbox" ng-model="isChecked" ng-click="tileCtrl.onCheckAllTiles(isChecked)" title="{{tileCtrl.settings.selectAllTitle}}" />
     </div>
-    <div class="miq-per-page col-md-2 col-ld-2">
+    <div class="miq-per-page col-md-2 col-ld-2 col-xs-3">
```

Comment:
`xs-3` but `md-2` .. indented or a typo? (Same on line 14)

Also, `col-ld-2` doesn't do anything, it's supposed to be `lg` I think ;) (all over this file..)